### PR TITLE
Fix deprecation check

### PR DESCRIPTION
**Pull request by tiagonapoli**
#### What is the purpose of this pull request?
Fix deprecation check.

#### What problem is this solving?
Updating the version to a non deprecated one from a deprecated one wouldn't release the deprecation lock.

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/deprecation-check && \
yarn && yarn watch
```
Change the `package.json` version to `2.90.1`. Run two commands, it should be enough to trigger the deprecation lock.
Change the `package.json` version to `2.90.0`. Run commands freely, it shouldn't trigger the deprecation lock in any time.

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [x] Update `CHANGELOG.md`

### CODE REVIEW

File: src/CLIPreChecker/DeprecationChecker/DeprecationChecker.test.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { DeprecationChecker } from './DeprecationChecker'
+import { IDeprecationCheckerStore, VersionDeprecationInfo } from './DeprecationCheckerStore'
+
+class DeprecationCheckerStoreMock implements IDeprecationCheckerStore {
+  public storeFilePath = 'mockFIlePath'
```
Comment (NataliaGodot) [inline src/CLIPreChecker/DeprecationChecker/DeprecationChecker.test.ts]: Should it be `mockFilePath`?

File: src/CLIPreChecker/DeprecationChecker/DeprecationChecker.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import chalk from 'chalk'
+import { spawn } from 'child_process'
+import { join } from 'path'
+import { DeprecationCheckerStore, IDeprecationCheckerStore, VersionDeprecationInfo } from './DeprecationCheckerStore'
+
+export class DeprecationChecker {
+  private static readonly DEPRECATION_CHECK_INTERVAL = 1 * 3600 * 1000
```
Comment (NataliaGodot) [inline src/CLIPreChecker/DeprecationChecker/DeprecationChecker.ts]: Did you intend to decrease the check interval besides fixing it?

Comment (tiagonapoli) [reply src/CLIPreChecker/DeprecationChecker/DeprecationChecker.ts]: Yeah, I though it woudn't hurt doing it one time per hour

File: src/CLIPreChecker/DeprecationChecker/DeprecationChecker.test.ts (hunk 1 above)
Comment (tiagonapoli) [reply src/CLIPreChecker/DeprecationChecker/DeprecationChecker.test.ts]: Yes! Will fix

**Pull request merged**